app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person_dict in people:
+        Person(person_dict["name"], person_dict["age"])
+
+    for person_dict in people:
+        person = Person.people[person_dict["name"]]
+        if "wife" in person_dict and person_dict["wife"] is not None:
+            person.wife = Person.people.get(person_dict["wife"])
+        if "husband" in person_dict and person_dict["husband"] is not None:
+            person.husband = Person.people.get(person_dict["husband"])
```

Comment:
Ensure that the related person (wife or husband) is created before attempting to set the relationship. If the related person is not in the `Person.people` dictionary, it will result in a `None` value being assigned, which may not be the intended behavior. Consider adding a check or creating the related person if they do not exist.